docs/examples/google-cloud-function.ts

```diff
@@ -2,12 +2,12 @@ import { Telegraf } from 'telegraf'
 
 const { BOT_TOKEN, PROJECT_ID, FUNCTION_NAME, REGION } = process.env
```

Comment:
You should check if these variables are set rather than disabling eslint warnings.
```ts
if (!BOT_TOKEN) {
  throw new Error('BOT_TOKEN required');
}
```

Having this check the first warning will be gone. Checking the other 3 and the second warning will be gone.

Typescript is there to help us, not to disable its warnings :)

Reply:
The solution in that case would end up being overly specific. If you don't provide a bot token, telegram already throws an error. And `PROJECT_ID`, `FUNCTION_NAME`, and `REGION` are provided as env variables by default in Google Cloud Functions

Reply:
Yes, I could check every env variable for undefined, but at that point, this example starts to become a little too complicated to be used as an example no?

Reply:
If you are sure they are there maybe just use the `!` then.

```ts
const bot = new Telegraf(BOT_TOKEN!)
```

Seems more descriptive to me than ignoring eslint warnings.